stan/math/prim/prob/bernoulli_cdf.hpp

```diff
@@ -65,21 +68,21 @@ return_type_t<T_prob> bernoulli_cdf(const T_n& n, const T_prob& theta) {
       continue;
     }
 
-    const T_partials_return Pi = 1 - theta_vec.val(i);
-
-    P *= Pi;
+    P += theta_vec.val(i);
```

Comment:
`theta_vec` should be called `log1m_theta_vec` because the former implies it's a vector view of `theta`, which is wrong.  This took me several minutes to sort out from reading just these diffs---I had to go back to the code to figure out that `theta_vec` as misnamed.  (And yes, I realize you didn't come up with that name.)